archive/rssi_validation/src/processing.py

```diff
@@ -1,4 +1,5 @@
 
```

Comment:
no shebang?

Reply:
This script is archived, but I can follow-up and add a shebang if desired. Definitely want to do that for actively-used scripts, though

Reply:
archive doesn't matter